app/services/salesforce_service/base.rb

```diff
@@ -8,9 +8,11 @@ class Base
     class_attribute :timeout
     class_attribute :error
     class_attribute :force
+    class_attribute :unit_data_tries
```

Comment:
the whole `unit_data_tries` is purely for the fake error, right? otherwise, this looks good 👍

Reply:
yep just for the error trigger